packages/api/src/checkTypes.manual.ts

```diff
@@ -96,7 +104,7 @@ async function tx (api: ApiPromise, keyring: TestKeyringMap): Promise<void> {
     });
 
   // with options and the callback
-  const nonce2 = await api.query.system.accountNonce<Index>(keyring.alice.address);
+  const nonce2 = await api.query.system.accountNonce(keyring.alice.address);
```

Comment:
Same here, it should be `Index`

Reply:
Just tried, even with `<Index>`, nonce2 is a u32. So related to type aliases